pandas/tools/pivot.py

```diff
@@ -106,6 +106,10 @@ def pivot_table(data, values=None, index=None, columns=None, aggfunc='mean',
         else:
             values_multi = False
             values = [values]
+        # GH14938 Make sure values are in data
+        for i in values:
+            if i not in data:
+                raise KeyError(i)
```

Comment:
@MaximilianR That won't let the user know the specific values that were missing.  Not sure if we want to raise a KeyError with a text of all the values, as it seems with other functions, pandas raises a KeyError for the first missing key.